src/__test__/components/ContentWrapper.test.jsx

```diff
@@ -10,11 +10,15 @@ import ContentWrapper from '../../components/ContentWrapper';
 const { Item } = Menu;
 
 jest.mock('next/router', () => ({
-  useRouter: jest.fn().mockImplementation(() => ({
-    query: {
-      experimentId: '1234',
-    },
-  })),
+  useRouter: jest.fn()
+    .mockImplementationOnce(() => ({ // 1st test
+      query: {
+        experimentId: '1234',
+      },
+    }))
+    .mockImplementationOnce(() => ({ // 2nd test
+      query: {},
+    })),
```

Comment:
This is done to simulate going to `/data-management`, where `experimentId` is undefined

Reply:
good idea!